auter.yumdnfModule

```diff
@@ -73,14 +73,14 @@ function apply_updates() {
       # Save the staged rpms to a variable
       RPMS="${DOWNLOADDIR}/${CONFIGSET}/*.rpm"
       # Check if the updates staged are still required
-      "${PACKAGE_MANAGER}" check-update $(rpm -qp ${RPMS}) &>/dev/null
+      "${PACKAGE_MANAGER}" check-update $(rpm -qp ${RPMS}) &>${DATADIR}/last-apply-output-${CONFIGSET}
       local RC=$?
     fi
     # When passing RPM files to dnf/yum, the update verb won't install any that aren't already
     # installed (i.e. dependencies of other packages). Instead we need to use install.
     UPDATEACTION="install"
   else
-    local RC=$(${PACKAGE_MANAGER} check-update ${PACKAGEMANAGEROPTIONS} &>/dev/null; echo $?)
+    local RC=$(${PACKAGE_MANAGER} check-update ${PACKAGEMANAGEROPTIONS} &>${DATADIR}/last-apply-output-${CONFIGSET}; echo $?)
```

Comment:
No longer relevant after proposed PR to switch to rpm for checking updates validity.